synthtool/gcp/templates/python_library/docs/summary_overview.md

```diff
@@ -0,0 +1,22 @@
+[
+This is a templated file. Adding content to this file may result in it being
+reverted. Instead, if you want to place additional content, use
+"overview_content.md" file in `docs/` directory. The Sphinx tool will
+pick up on the content and merge the content.
+]: #
+
+# {{ metadata['repo']['name'] }} API
+
+Overview of the APIs available for {{ metadata['repo']['name'] }} API.
+
+## All entries
+
+Take a look at all classes, methods and properties & attributes for
+{{ metadata['repo']['name'] }} API.
+
+[classes]({{ metadata['repo']['client_documentation'] }}/summary_class.html)
+
+[methods]({{ metadata['repo']['client_documentation'] }}/summary_method.html)
+
+[properties and
+attributes]({{ metadata['repo']['client_documentation'] }}/summary_property.html)
```

Comment:
These files don't exist yet. Should we consider publishing them before we add an overview page?

Reply:
They were planned to be simultaneously published, but I've released new version of the plugin.